of-builder/main.go

```diff
@@ -154,6 +154,18 @@ func build(w http.ResponseWriter, r *http.Request) ([]byte, error) {
 		insecure = val
 	}
 
+	frontendAttrs := map[string]string{
+		"source": cfg.Frontend,
+	}
+
+	if val, exists := os.LookupEnv("http_proxy"); exists {
```

Comment:
Let's read it in the entry point and pass it to the function as a series of build args